lib/space.js

```diff
@@ -34,7 +34,10 @@ const spaceMethods = [
 
   'processAsset',
 
-  'getUsers'
+  'getUsers',
+
+  'createUpload',
```

Comment:
1. You could group these methods with `processAsset` as they belong to the same category
2. Add the new methods to the list in `test/unit/space.js`